doc/source/ray-air/examples/dreambooth_finetuning.rst

```diff
@@ -147,30 +147,30 @@ We ran training using 2, and 4 workers/GPUs to compare throughput.
 
 Setup:
 
-* One g5.12xlarge nodes with 4 A10G GPUs each
+* One g5.12xlarge nodes with 4 A10G GPUs
 * Model as configured below
 * Data from this example
 * 200 regularization images
 * Training for 4 epochs (800 steps)
+* 3 runs per configuration
```

Comment:
Do we have std deviation / error bar for the plot?

Reply:
not much value to the visual. std dev is super small and almost not visible.